scripts/automation/publish.py

```
@@ -0,0 +1,142 @@
+    if not os.path.isdir(os.path.join(pages_repo, ".git")):
+        print(f"ERROR: git リポジトリではありません: {pages_repo}", file=sys.stderr)
+        return 2
+
+    number = derive_number(manual_dir, args.number)
+    rel_path = os.path.join("manuals", number, "index.html")
+    dest = os.path.join(pages_repo, rel_path)
+    os.makedirs(os.path.dirname(dest), exist_ok=True)
+    shutil.copy2(src_html, dest)
+
+    size_kb = os.path.getsize(dest) // 1024
+    print(f"=== 配置 ===")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)